src/components/table/README.md

```diff
@@ -22,7 +22,7 @@ More information about when to use table can be found on [GOV.UK Design System](
 
         <tr class="govuk-c-table__row">
 
-          <td class="govuk-c-table__cell" scope="row">January</td>
+          <td class="govuk-c-table__cell">January</td>
```

Comment:
Can I get a review on this change? I'm not super savvy on table semantics.

https://dequeuniversity.com/rules/axe/1.1/scope